src/WebJobs.Script/Binding/DocumentDBScriptBindingProvider.cs

```diff
@@ -43,7 +46,19 @@ public override bool TryCreate(ScriptBindingContext context, out ScriptBinding b
         /// <inheritdoc/>
         public override void Initialize()
         {
-            Config.UseDocumentDB();
+            DocumentDBConfiguration documentDBConfiguration = new DocumentDBConfiguration();
+
+            JObject configSection = (JObject)Metadata.GetValue("documentDB", StringComparison.OrdinalIgnoreCase);
```

Comment:
We're getting into a little of the weirdness of the "CosmosDB" vs "DocumentDB" naming conventions here. This global host.json setting isn't strictly required, is it? The lease options can be set in function.json for each function, if needed. I wonder if we just leave this off to keep things simpler and we can always re-add it in the future.